### PR TITLE
Remove PDone

### PR DESCRIPTION
**Pull request by aaltay**
@chamikaramj @robertwb What do you think about this?

### CODE REVIEW

File: sdks/python/apache_beam/runners/dataflow/native_io/iobase.py [hunk 1]
```diff
@@ -315,4 +313,4 @@ def __init__(self, sink):
 
   def expand(self, pcoll):
     self._check_pcollection(pcoll)
-    return pvalue.PDone(pcoll.pipeline)
+    return pvalue.PValue(pcoll.pipeline)
```
Comment (robertwb) [inline sdks/python/apache_beam/runners/dataflow/native_io/iobase.py]: Perhaps they don't need to return anything at all (i.e. None)?

Comment (aaltay) [inline sdks/python/apache_beam/runners/dataflow/native_io/iobase.py]: Makes sense, updated to return `None`.

**Comment by chamikaramj** [comment]
LGTM

**Comment by aaltay** [comment]
@robertwb PTAL

**Comment by robertwb** [comment]
I discovered an issue with the missing PDone: without an output, we can't figure out whether an operation without inner operations is "primitive" or not. Composite operations are not visited by runners.

**Comment by aaltay** [comment]
Should we keep `PDone` then? Or how about returning an empty `PCollection` from `expand`?

**Comment by robertwb** [comment]
This may all go away once Native Writes are never injected directly into
the graph at construction time, but instead replaced by runners after the
fact.

**Comment by aaltay** [comment]
I will close this PR. Opened https://issues.apache.org/jira/browse/BEAM-1885 for tracking it.

**Pull request closed without merging**